client/components/plans/plans-compare/index.jsx

```diff
@@ -96,6 +99,39 @@ var PlansCompare = React.createClass( {
 		}, this );
 	},
 
+	showFreeTrialException: function() {
+		const hasTrial = this.props.sites ? this.props.sites.getSelectedSite().plan.free_trial : false,
+			canStartTrial = this.props.sitePlans ? this.props.sitePlans.data.some( function( plan ) {
```

Comment:
Not necessary to change, but we have used [`_.property`](https://lodash.com/docs#property) for this in the past:

```js
canStartTrial = this.props.sitePlans
  ? this.props.sitePlans.data.some( property( 'canStartTrial' ) )
  : false;
```